pkgs/by-name/fl/fluidsynth/package.nix

```
@@ -0,0 +1,72 @@
+{
+  stdenv,
+  darwin ? { },
+  lib,
+  fetchFromGitHub,
+  buildPackages,
+  pkg-config,
+  cmake,
+  alsa-lib,
+  glib,
+  libjack2,
+  libsndfile,
+  libpulseaudio,
+}:
+
+stdenv.mkDerivation rec {
```

Use `finalAttrs` pattern

```suggestion
stdenv.mkDerivation (finalAttrs: {
```